src/TranslationLoaders/TranslationLoader.php

```diff
@@ -12,5 +12,5 @@
      *
      * @return array
      */
-    public function loadTranslations(string $locale, string $group): array;
```

Comment:
This is a breaking change. If these were to be merged, all users of this package would receive errors that the function signature has been changed. Make that parameter optional.

Reply:
Even if I make it optional, it is still a breaking change.

Reply:
indeed...